### DISCUSSION TITLE
Library zxing removed from vcpkg?

### DISCUSSION THREAD

**Discussion by tobecker** [microsoft/vcpkg#23769] [General]
Hello,

I updated vcpkg to the latest release today and found that zxing-cpp is no longer present in the ports folder.

Has the library been deleted or will it be included again?
On the website https://vcpkg.info/ the library is still listed.

Regards

**Comment by autoantwort** [comment]
Yeah the port was replaced by `nu-book-zxing-cpp`, see https://github.com/microsoft/vcpkg/pull/22657#pullrequestreview-875487205
